services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -71,7 +70,8 @@ public AccountConsole(RealmModel realm, ClientModel client, Theme theme) {
         this.realm = realm;
         this.client = client;
         this.theme = theme;
-        this.authManager = new AppAuthManager();
+        //this.authManager = new AppAuthManager();
+        this.authManager = null;
```

Comment:
Was this a mistake? Seems irrelevant and a breaking change I think.

Reply:
Temporary code I was using but that I forgot to remove before my commit